.github/workflows/release-integration.yml

```diff
@@ -0,0 +1,79 @@
+name: Release Integration Test Workflow
```

Comment:
Is this needed? Can we just modify existing test to run on release branch PR?

Reply:
The only difference here is that we actually pack the tarball and use the tarball instead of `file: ../../` (the regular CI Workflow doesnt pack the tarball)

We could potentially replace our existing CI workflow to use a generated tarball by default.